docs/_index.yaml

```diff
@@ -4,25 +4,30 @@ description: <!--no description-->
 landing_page:
   custom_css_path: /site-assets/css/style.css
   rows:
-  - heading: TensorFlow's visualization toolkit
+  - heading: "TensorBoard: TensorFlow's visualization toolkit"
     items:
     - classname: devsite-landing-row-50
       description: >
         TensorBoard provides the visualization and tooling needed for 
-        machine learning experimentation. It enables:
+        machine learning experimentation:
         <ul style="padding-left: 20px; margin: 20px 0;">
           <li>Tracking and visualizing metrics such as loss and accuracy</li>
           <li>Visualizing the model graph (ops and layers)</li>
-          <li>Viewing histograms and distributions of Tensors (e.g., weights 
-          and biases) to track how they change over time</li>
+          <li>Viewing histograms of weights, biases, or other tensors as they
+              change over time</li>
           <li>Projecting embeddings to a lower dimensional space</li>
           <li>Displaying images, text, and audio data</li>
           <li>Profiling TensorFlow programs</li>
           <li>And much more</li>
         </ul>
-
       image_path: /tensorboard/images/tensorboard.gif
-
+      buttons:
+      - label: Learn More
```

Comment:
Just wondering, what's the rule for putting quotes? Do these need to be quoted like the titles?

Reply:
This is YAML; quotes are roughly optional (subject to a ton of caveats,
because YAML).